edsdme/blocks/search-full/SearchCardsStyles.js

```diff
@@ -30,13 +30,19 @@ export const searchCardsStyles = css`
     border: 1px solid #BCBCBC;
     border-radius: 12px;
   }
+
+  .partner-cards {
```

Comment:
@cod17828 this impacts all partner cards. Are the 269px as per a design or are they what works? it seems to be strange to hardcode this value :-)